vajram/src/main/java/com/flipkart/krystal/vajram/inputs/resolution/SimpleInputResolver.java

```diff
@@ -91,6 +91,11 @@ public DependencyCommand<Inputs> resolve(
                                   new ValueOrError<>(o, Optional.empty()))))
                   .toList());
       }
+    } catch (Exception e) {
+      return skipExecution(
+          String.format(
```

Comment:
Should we add exception name and message to this string as well, for better debugging?